docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -43,8 +43,7 @@ <h2>Release notes: Open GEE 5.2.5</h2>
   <h5>
     <p id="overview_5.2.5">New Features</p>
   </h5>
-    <p><strong>C++98 no longer supported</strong>. As of 5.2.4 C++11 syntax can be used and building open GEE using C++98 standard will no longer be supported</p>
-
+  <p>Server definitions and other information can now be returned in JSON format. The <code>v</code> parameter can be added to query string to specify which version of the API will be returned. The valid options are <code>v=1</code> and <code>v=2</code>. Version one returns data in the current format. Version two returns data in JSON format. An example url which returns data in JSON format looks like: <code>http://yourserver.org/virtual-server-name/query?request=Json&amp;v=2</code>.</p>
```

Comment:
bold the first sentience using the `<strong>` tag like the one you replaced did.

Reply:
may be for that first phrase use: <p><strong>When a GEE server is queried for Server Definitions the returned response is a valid JSON </string> ..... </p>

Reply:
in the given example at the end of the url the database publish entry need to be appended. So the request would be:
http://<GEE earth server>/<database publish point>/query?request=Json&amp;v=2
-->
http://yourserver.org/virtual-server-name/database_publish_point/query?request=Json&amp;v=2